### PR TITLE
06_Edith_Afaf Himmatul Ulum

### PR DESCRIPTION
**Pull request by Faa238**

### CODE REVIEW

**Comment by Faa238** [comment]
Modify the hyperparameters to train CNN (learning_rate and momentum) in the pytroch-mnist.ipynb so that the accuracy result
reaches 97%
